src/pages/verify/e-ids/norwegian-bankid.mdx

```diff
@@ -77,29 +79,32 @@ This can be fixed by ordering a new netcentric BankID:
 ![Order new BankID](./images/no-bank-id-order-new-bankid.png)
 
 ### Testing BankID Biometric
+
 Start by creating a netcentric test user as described above, and add an email and a phone number to the user.
```

Comment:
Adding phone and email is now already described above in the Netcentric section (the new step 5 above).

Perhaps move that to this (Biometric) section ?

Reply:
Right! Thanks